tests/snapshots/tests.integrations.laminas.v1_4.common_scenarios_test.test_scenario_get_return_string.json

```diff
@@ -11,7 +11,7 @@
       "_dd.p.dm": "-1",
       "component": "laminas",
       "http.method": "GET",
-      "http.route": "simple",
+      "laminas.route.name": "simple",
```

Comment:
Can we really plainly remove this `http.route` tag? Or is it gonna be set back in a future PR? (Applies to Laminas changes)

Reply:
Not sure about this. @bwoebi what's your view?

Reply:
A future PR will give it a semantically different value, so it's fine to rename this.